oak-commons/src/main/java/org/apache/jackrabbit/oak/commons/collections/package-info.java

```diff
@@ -21,7 +21,7 @@
  * Utilities for Java collections and streams.
  */
 @Internal(since = "1.0.0")
-@Version("1.2.0")
+@Version("1.3.0")
```

Comment:
This shouldn't be required.
Could you please create PR without updating the baseline version?

Reply:
Done.